src/ramses_tx/parsers.py

```diff
@@ -2128,6 +2128,18 @@ def parser_31d9(payload: str, msg: Message) -> dict[str, Any]:
     }
 
     if msg.len == 3:  # usu: I -->20: (no seq#)
+        if msg.src.type == "32" and payload[:4] == "0000":  # Vasco D60 HRU
+            # can't access 'vasco' scheme as this PR is separate from _22f1
```

Comment:
??? See above

Reply:
This is not directly connected to the above question. Once the 22F1 is merged I can point this parser to the `vasco` scheme.